src/structlog/processors.py

```diff
@@ -407,11 +409,9 @@ def __init__(
     def __call__(
         self, logger: WrappedLogger, name: str, event_dict: EventDict
     ) -> EventDict:
-        exc_info = event_dict.pop("exc_info", None)
-        if exc_info:
-            event_dict["exception"] = self.format_exception(
-                _figure_out_exc_info(exc_info)
-            )
+        exc_info = _figure_out_exc_info(event_dict.pop("exc_info", None))
+        if exc_info != (None, None, None):
```

Comment:
It seems like this would pass a `None` exc_info in the event dict into the formatter. Is this expected?

Reply:
Changed it to `if exc_info and exc_info != (None, None, None)`. This should work.

Not happy though, that `exc_info` can be literally anything.  I will take a look if this can somehow be changed/improved.